### PR TITLE
|RC1] nit: throughout -> throughput

### PR DESCRIPTION
**Pull request by collivier**
Co-Authored-By Rihab Banday <rihab.banbay@ericsson.com>

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

File: doc/ref_cert/RC1/chapters/chapter02.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ This chapter mainly covers the overall requirement for Reference Conformance tes
 
 <!---Validations is used to indicate testing performed to confirm the actual output of a product meets the expected, or desired outcome, or behavior.--->
 
-All Terms utilized throughout this chapter are intended to align with CVC definitions, and their use through CVC documentation, guidelines, and standards. This chapter will outline the Requirements, Process, and Automation, needed to deliver NFVI conformance.
```
Comment (gkunz) [inline doc/ref_cert/RC1/chapters/chapter02.md:40]: This instance of "thoughout" is actually correct. :-)

**Review by gkunz — CHANGES_REQUESTED** [review]

File: doc/ref_cert/RC1/chapters/chapter02.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ This chapter mainly covers the overall requirement for Reference Conformance tes
...
+All Terms utilized throughput this chapter are intended to align with CVC definitions, and their use through CVC documentation, guidelines, and standards. This chapter will outline the Requirements, Process, and Automation, needed to deliver NFVI conformance.
```
Comment (pgoyal01) [inline doc/ref_cert/RC1/chapters/chapter02.md:40]:
```suggestion
All Terms utilized throughout this chapter are intended to align with CVC definitions, and their use through CVC documentation, guidelines, and standards. This chapter will outline the Requirements, Process, and Automation, needed to deliver NFVI conformance.
```

File: doc/ref_cert/RC1/chapters/chapter02.md (hunk 1 above)
Comment (collivier) [reply doc/ref_cert/RC1/chapters/chapter02.md:40]: oh I failed :)

**Comment by collivier** [comment]
never trust sed :)

**Review by pgoyal01 — APPROVED** [review]

**Review by rihabbanday — APPROVED** [review]

**Review by gkunz — APPROVED** [review]

**Pull request merged**